frame/staking/src/benchmarking.rs

```diff
@@ -634,6 +644,81 @@ benchmarks! {
 	verify {
 		assert!(!Validators::<T>::contains_key(controller));
 	}
+
+	rebag {
+		// The most expensive case for this call:
+		//
+		// - It doesn't matter where in the origin bag the stash lies; the number of reads and
+		//   writes is constant. We can use the case that the stash is the only one in the origin
+		//   bag, for simplicity.
+		// - The destination bag is not empty, because then we need to update the `next` pointer
+		//   of the previous node in addition to the work we do otherwise.
+
+		use crate::voter_bags::{Bag, Node};
+
+		let make_validator = |n: u32, balance_factor: u32| -> Result<(T::AccountId, T::AccountId), &'static str> {
+			let (stash, controller) = create_stash_controller::<T>(n, balance_factor, Default::default())?;
+			whitelist_account!(controller);
+
+			let prefs = ValidatorPrefs::default();
+			// bond the full value of the stash
+			let free_balance = T::Currency::free_balance(&stash);
+			Staking::<T>::bond_extra(RawOrigin::Signed(stash.clone()).into(), free_balance)?;
```

Comment:
hmm how does this work? if the account is not bonded, how can you call `bond_extra`?

Reply:
`create_stash_controller` does bond the stash account, but only 10% of it. This bonds its full value.